drivers/bridge/bridge.go

```diff
@@ -315,6 +300,18 @@ func (c *networkConfiguration) fromMap(data map[string]interface{}) error {
 	return nil
 }
 
+func (n *bridgeNetwork) getDriverChains() (*iptables.ChainInfo, *iptables.ChainInfo, error) {
+	n.Lock()
```

Comment:
if you lock n for the whole function, then you do not need to get a reference of n.driver.
Otherwise you would lock n only for the time needed to get the reference d.